crates/cli/src/main.rs

```diff
@@ -186,6 +189,16 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
+    let mut node_service_tasks: Vec<Pin<Box<dyn Future<Output = anyhow::Result<()>>>>> = Vec::new();
+    let l1_sidecar = match config.l1_anvil_port {
+        Some(l1_anvil_port) => {
+            let (l1_sidecar, l1_sidecar_runner) = L1Sidecar::builtin(l1_anvil_port).await?;
+            node_service_tasks.push(Box::pin(l1_sidecar_runner.run()));
+            l1_sidecar
+        }
+        None => L1Sidecar::none(),
+    };
```

Comment:
Do we want L1-L2 support enabled by default rather then be toggled via l1-port option? Perhaps this is intentional right now?

Reply:
Agreed that it should be disabled by default. Went with options as per @popzxc's suggestion for now. Also I put them under "UNSTABLE - L1" header so hopefully it's clear enough that those are subject to change.